templates/traefik-forward-auth.yaml

```diff
@@ -9,7 +9,6 @@ spec:
   requires:
     matchLabels:
       kubeaddons.mesosphere.io/name: dex
-      values.chart.helm.kubeaddons.mesosphere.io/traefik-forward-auth: "https://raw.githubusercontent.com/mesosphere/charts/6c43b8ab10108fb1adba5c6dd10e800e5f1abdd0/staging/traefik-forward-auth/values.yaml"
```

Comment:
think we want to keep this for `kommander` work.

Reply:
resolving because we're simply trying to match history.

Reply:
yeah,  i just cherry-pick the change from here: https://github.com/mesosphere/kubeaddons-configs/pull/229/files